docs/pages/home/home.yaml

```diff
@@ -332,6 +332,92 @@ paths:
       security:
         - VesselAPIToken: []
     parameters: []
+  /api/passthrough:
+    post:
+      summary: Passthrough Request
+      operationId: passthrough
+      parameters:
+      - name: x-vessel-access-token
+        in: header
+        description: Access Token
+        required: true
+        schema:
+          type: string
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                title: responseBody
+                description: ''
+                type: object
+                properties:
+                  headers:
+                    type: object
+                  path:
+                    type: string
+                    description: The full URL that the request was sent to.
+                  statusCode:
+                    type: number
+                    description: The HTTP status code of the response from the downstream
+                      system.
+                  body:
+                    AnyValue:
+                      description: The body of the response from the downstream system.
+                required:
+                - id
+              examples:
+                example-1:
+                  value:
+                    headers:
+                      content-type: application/json
+                    url: https://sometool.com/api/v1/endpoint
+                    statusCode: 200
+                    body:
+                      id: 123123123123
+                      success: true
+      requestBody:
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                method:
+                  type: string
+                  enum:
+                  - GET
+                  - POST
+                  - PUT
+                  - PATCH
+                  - DELETE
+                path:
+                  type: string
+                  description: The path to send the request to. Vessel handles the
+                    domain, for example, a request to Salesforce would use `/services/data/v52.0/sobjects/Account`.
+                    Full URLs are also supported, e.g. `https://mydomain.my.salesforce.com/services/...`.
```

Comment:
just confirming this is true? I wasn't sure what the resolution was from this morning

Reply:
It should be path only. Full urls are not supported

Reply:
They're supported in the client but the endpoint prevents it now I believe

Reply:
Yup, the endpoint has validation that prevents full path

Reply:
Yeah, just confirmed this locally, it works with both — regarding whether or not that's what we want, I'm not sure. I feel like having the flexibility makes sense

Reply:
got it, updated the description